### DISCUSSION TITLE
How to show the line number where an error occurred

### DISCUSSION THREAD

**Discussion by joshuafolkken** [MikeSchulze/gdUnit4#565] [Q&A] [answered]
When I throw an error with this code, the line number coming up as N/A. What should I do?

# Unit Test Code

```gd
extends GdUnitTestSuite

func test_a() -> void:
	assert_bool(true).is_false()
```

# gdUnitConsole
```sh
FALIED ... line <n/a>: Expecting: "false" but is 'true'
```
![Screenshot 2024-09-04 at 10 40 28](https://github.com/user-attachments/assets/5f19c95e-5738-4c6f-8ef2-a735be162299)

# Environment Details
- GdUnit4 4.3.4
- Godot 4.3, stable, Typed GDScript
- macOS Sonoma 14.6.1

**Comment by MikeSchulze** [comment, marked as answer]
You need to run the tests in debug mode. 
Godot do not provide a stack trace at runtime to grab the line number.
Check the GdUnit4 documentation https://mikeschulze.github.io/gdUnit4/testing/run-tests/
![image](https://github.com/user-attachments/assets/60fc09af-c730-4f86-876a-18fbacd6dae4)

**Reply by joshuafolkken** [reply]
Thank you! Totally missed that.
